style.css

```diff
@@ -50,17 +54,24 @@ main {
 	/* Initially hidden, will be shown when JS plays the search animation */
 	opacity: 0;
 	transition: opacity 0.5s ease;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	transition: opacity 0.25s ease;
 }
 
+
 input[type="text"] {
 	padding: 10px;
 	font-size: 16px;
-	border: 1px solid #ccc;
+	border: 1px solid var(--bright-bg-color);
 	border-radius: 5px;
+	box-sizing: border-box;
+	background-color: var(--primary);
 }
 
-input.query {
-	width: 50%;
+#query {
+	width: 100%;
```

Comment:
Nice. I wasn't aware that this actually didn't do anything since having changed classes to ids :)

Initially, I wanted the query input to span half of the page (so `50vw`), but this is actually to large for large screens and to small for small screens. It seems that Google uses `max-width: 584px;` instead.

Reply:
I think the use of vh can be problematic because there seem to be some problems related to it with mobile browsers. Therefore I would use max-width instead.